freetextresponse/freetextresponse.py

```diff
@@ -297,7 +297,20 @@ def student_view(self, context={}):
                 'other_responses': [],
             }
         )
-        template = get_template('freetextresponse_view.html')
+
+        """
```

Comment:
Multiple doc strings should be set under the function definition, otherwise please use comments.